packages/web/src/scss/components/UNSTABLE_Avatar/README.md

```diff
@@ -0,0 +1,81 @@
+# UNSTABLE Avatar
+
+⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+Please use it with caution.
+
+The `UNSTABLE_Avatar` component is used to represent a user or entity.
+
+It can be a circle or a square (with rounded corners, using `radius-100` token) and
+can have different sizes.
+
+## Example
+
+```html
+<div class="UNSTABLE_Avatar UNSTABLE_Avatar--medium" title="Jiří Bárta">JB</div>
```

Comment:
Maybe we should use some general names like `John Doe` or something similar.

Reply:
:D Can be. I was inspired by Header, where we use Marian's name as he was the designer behind the component :) But there can be anything :)

Reply:
Ok, I know. But I would like to avoid real names in the demo or elsewhere.